src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+input {
+  outline: none;
+}
+
+::placeholder {
+  font-family: AvenirBook, sans-serif;
+  color: #3d4e61;
+}
+
+.search-bar {
+  width: 100%;
+  display: inline-block;
+  position: relative;
+}
+
+.image {
+  position: absolute;
+}
+
+.field {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: AvenirHeavy, sans-serif;
+  font-weight: 900;
+  color: #3d4e61;
+  margin-top: 20px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.bigger__search-image {
+  top: 45px;
+  left: 26px;
+}
+
+.bigger__search-field {
+  padding: 25px 0 23px 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bigger__search-field::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
Write all styles to generic class, and then override sizes for small input

Big input here - is like default state of input, and small - is a modified default input, smaller version of default input